src/KeyVault/KeyVault/Commands/NewAzureKeyVault.cs

```diff
@@ -117,76 +125,143 @@ public class NewAzureKeyVault : KeyVaultManagementCmdletBase
         [Parameter(Mandatory = false, HelpMessage = "Specifies the network rule set of the vault. It governs the accessibility of the key vault from specific network locations. Created by `New-AzKeyVaultNetworkRuleSetObject`.")]
         public PSKeyVaultNetworkRuleSet NetworkRuleSet { get; set; }
 
+        /// <summary>
+        /// ApiVersion
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   HelpMessage = "Specifies the SDK api version. Default value is '2023-02-01'.")]
+        public string ApiVersion { get; set; } = "2023-02-01";
```

Comment:
we should not allow customer to specify api version in general. Any special reason?